examples/dask-nvtabular-criteo-benchmark.py

```diff
@@ -20,9 +20,11 @@
 import time
 import warnings
 
+import boto3
 import rmm
 from dask.distributed import Client, performance_report
 from dask_cuda import LocalCUDACluster
+from google.cloud import storage
```

Comment:
`boto3` and `google.cloud` shouldn't be dependencies to run the benchmark.  Need a `try`/`except` for optional libraries.